client/src/components/GraphViewerComponent/GraphViewerRelationshipCreateComponent/GraphViewerRelationshipCreateComponent.js

```diff
@@ -164,16 +163,22 @@ export class GraphViewerRelationshipCreateComponent extends Component {
               errorMessage={hasRequiredRelError ? "Please select a relationship" : null}
               onChange={this.onSelectedRelChange} />
           </div>
-        } {
-          !hasRelationships && <div className="warning-icon-wrapper">
-            <div className="warning-icon">
-              <IconButton iconOnly="true" iconProps={{ iconName: warningIconName }} title="Warning Relationship" ariaLabel="Warning  Relationship" />
-            </div>
-            <div className="warning-text">
-              <Label className="warning-label">No relationship available, try swapping source and target</Label>
-            </div>
-          </div>
         }
+        { !hasRelationships && <MessageBar
+          aria-label="Warning  Relationship"
+          className="ms-MessageBar-Relationship"
+          messageBarType={MessageBarType.error}
+          role="alert"
+          styles={{
+            root: {
+              position: "absolute",
+              top: 0,
+              zIndex: 1
+            }
+          }}
```

Comment:
I think this is causing it to be at the top of the modal instead of in its original position.  Is it possible to add these styles to the CSS class?

Reply:
Already Fix the position using CSS